packages/ds-healthcare-gov/src/components/index.js

```diff
@@ -11,9 +11,13 @@
 
 import './Button';
 
+import { setErrorPlacementDefault } from '@design-system';
```

Comment:
[question] Can you explain again why it's `@design-system` here instead of `@cmsgov/design-system` like on line 16?

Reply:
Doh! It shouldn't be. Good catch. It's copypasta.

Reply:
The docs site builder has a `@design-system` alias that always points to the current design system package (core or child). When I tested this after remaking-this change in our package, I didn't test it in a downstream app, and it just happens to work in my local docs site because of this alias.

Reply:
Ah, thanks. From the pairing session, I wasn't sure if one was needed vs the other in this situation. Thanks for resolving